### PR TITLE
Fix che-theia/generator tests inside container without git user.name and user.email

### PR DESCRIPTION
**Pull request by AndrienkoAleksandr**
### What does this PR do?
Fix che-theia/generator tests inside container without git user.name and user.email

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/14056

Signed-off-by: Oleksandr Andriienko <oandriie@redhat.com>

### CODE REVIEW

**Comment by musienko-maxim** [comment]
Can one of the admins verify this PR?

File: generator/tests/init-sources/init-sources.spec.ts [hunk 1]
```diff
@@ -65,6 +65,8 @@ describe("Test Extensions", () => {
 
     function initGit(path: string) {
         cp.execSync('git init', { cwd: path });
+        cp.execSync('git config --local user.name "test test"', { cwd: path });
+        cp.execSync('git config --local user.email test@gmail.com', { cwd: path });
```
Comment (benoitf) [inline generator/tests/init-sources/init-sources.spec.ts]: maybe better to use `example.com` or `eclipse.org` than `gmail` there

Comment (AndrienkoAleksandr) [reply generator/tests/init-sources/init-sources.spec.ts]: Done

Comment (benoitf) [reply generator/tests/init-sources/init-sources.spec.ts]: @AndrienkoAleksandr I think email should be still with pattern something@something (like `user@example.com`)

**Pull request merged**